lib/private/Metadata/Provider/ExifProvider.php

```diff
@@ -68,7 +68,12 @@ public function execute(File $file): array {
 		$size->setMetadata([]);
 
 		if (!$data) {
-			$sizeResult = getimagesizefromstring($file->getContent());
+			try {
+				$sizeResult = getimagesizefromstring($file->getContent());
+			} catch (\Throwable $ex) {
+				$this->logger->warning("Couldn't get image for ".$file->getId(), ['exception' => $ex]);
```

Comment:
Sane as in https://github.com/nextcloud/server/pull/36420/files
```suggestion
				$sizeResult = @getimagesizefromstring($file->getContent());
			} catch (\Exception $ex) {
				$this->logger->info("Couldn't get image for ".$file->getId(), ['exception' => $ex]);
```